remote-config.jsonc

```
@@ -153,7 +150,7 @@
           "message": "`ddev config --update` will change your project config where possible based on the code that is discovered in your project."
         },
         {
-          "message": "Don't forget that `ddev launch` can take a URI, like `ddev launch /admin/reports/status` and don't forget `ddev mailpit` to get the Mailpit UI."
+          "message": "Don't forget that `ddev launch` can take a URI or full URL, like `ddev launch /admin/reports/status` and don't forget `ddev mailpit` to get the Mailpit UI."
```

```suggestion
          "message": "Don't forget that `ddev launch` can take a URI or full URL, like `ddev launch /admin/reports/status`, and `ddev mailpit` gets you the Mailpit UI."
```
having "dont forget" twice within one sentence feels a bit much. how about the following?